cuizinart/metadata_schema.py

```diff
@@ -82,6 +97,17 @@ def __repr__(self):
         return '<Issue {!r}>'.format(self.issue_id)
 
 
+class NCFile(db.Model):
+    file_id = db.Column(db.Integer, primary_key=True)
+    file_name = db.Column(db.String, nullable=False)
```

Comment:
Should we add a unique constraint on `file_name`?

Reply:
Yes, you are correct.